deploy/kubernetes/README.md

```diff
@@ -1,18 +1,37 @@
-1. One-time per project: Create GCP service account for CSI driver
-    1. Export environment variables for location for service account private key file and name of the service account
+1. [One-time per project] Create GCP service account credential for CSI Driver and download JSON:
+    1. Go to https://pantheon.corp.google.com/apis/credentials
+    2. Select `Create credentials` -> ~Service account key`
+    3. For `Service Account` select `Compute Engine Default service account`
+    4. Select `JSON`
+    5. Click `Create` to download JSON for the private key to your local machine.
+    6. Rename the downloaded JSON file to `cloud-sa.json`
+    6. Export environment variables for 1) location of service account private key JSON file, 2) name you want of service account to be created by `setup_project.sh`, and 3) GCP project:
     ```
-    $ export SA_FILE=~/.../cloud-sa.json
-    $ export GCEPD_SA_NAME=sample-service-account
+    $ export SA_FILE=~/Downloads/cloud-sa.json
+    $ export GCEPD_SA_NAME=gcepd-csi-service-account
+    $ export PROJECT=my-gcp-project
     ```
-    2. Setup project with script
+    7. Setup project with script
     ```
     $ ./deploy/setup_project.sh
     ```
 2. Deploy driver to Kubernetes cluster
 ```
 $ ./deploy/kubernetes/deploy_driver.sh
 ```
-3. Create example PVC and Pod
-```
-$ kubectl apply -f ./examples/demo-pod.yaml
-```
\ No newline at end of file
+3. Create example workload to verify
+    1. Deploy yaml for PVC and Pod
+    ```
+    $ kubectl apply -f ./examples/demo.yaml
```

Comment:
`./examples/kubernetes/demo.yaml` ?

Also we can take the demopod from `./deploy/kubernetes/demo-pod.yaml`